cmd/pow/cmd/data/summary/summary.go

```diff
@@ -22,20 +29,33 @@ var Cmd = &cobra.Command{
 		c.CheckErr(err)
 	},
 	Run: func(cmd *cobra.Command, args []string) {
-		ctx, cancel := context.WithTimeout(context.Background(), c.CmdTimeout)
-		defer cancel()
-
 		var cids []string
 		if len(args) > 0 {
 			cids = strings.Split(args[0], ",")
 		}
 
-		res, err := c.PowClient.Data.CidSummary(c.MustAuthCtx(ctx), cids...)
-		c.CheckErr(err)
+		json := viper.GetBool("json")
 
-		json, err := protojson.MarshalOptions{Multiline: true, Indent: "  ", EmitUnpopulated: true}.Marshal(res)
-		c.CheckErr(err)
+		if json {
+			res, err := getSummary(cids)
+			c.CheckErr(err)
 
-		fmt.Println(string(json))
+			json, err := protojson.MarshalOptions{Multiline: true, Indent: "  ", EmitUnpopulated: true}.Marshal(res)
+			c.CheckErr(err)
+
+			fmt.Println(string(json))
```

Comment:
Nit/Style: maybe `return` and avoid the `else` indentation below. It's a personal taste though, so feel free to dismiss. I prefer avoiding else's if possible.

Reply:
Yea I'm always torn about that. Will delete the `else`.